vocabularies/functions.ttl

```diff
@@ -17,6 +17,8 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 cs:
     a skos:ConceptScheme ;
     dcterms:created "2021-06-16"^^xsd:date ;
+    vann:preferredNamespacePrefix "geof"^^xsd:string ;
```

Comment:
Don't we need a new PREFIX entry for vann

Reply:
Correct. I forgot it in this particular file. I added it now